### ISSUE TITLE
Job overview

### ISSUE THREAD

**Issue by 0xERR0R** [mcuadros/ofelia#122]
I plan to use ofelia for my deployments. I have some jobs, which runs periodically. I want to receive mail if a job fails. I would like to receive an job overview via mail (for example once weekly) with all executed jobs and their status (for example: jobname, date, status success/failed).

Can I somehow query for executed jobs in a certain time period? Maybe someone has an idea, how can I achieve it?

Btw. Nice project and great idea!

**Comment by Trane9991** [comment]
No, currently you cannot query or mail the report periodically. 

You can send emails or slack messages on each execution or each failure.

Another option is to save each execution report to the disk and then create ofelia periodic job (weekly, for example) and make it pick up that reports and send them over to you, however, I imagine that you would need to write some script for that.

**Comment by 0xERR0R** [comment]
Hmm. It sounds like a plan... I'll try it, thank you!

**Comment by Trane9991** [comment]
If you succeed in implementing that solution, would be nice if you can share it here, so other people can reuse it in the future :)
